src/Validator/RequiredMembers.php

```diff
@@ -37,19 +37,29 @@ class RequiredMembers extends AbstractValidator implements HydratorAwareInterfac
         self::MSG_EMPTY_MEMBERS   => 'Members "%s" within %s are required and therefore cannot be empty',
     ];
 
+    protected $className;
```

Comment:
I'd prefer it if this member was instead just passed as an parameter to validate(...). A classname in itself has nothing to do with the RequiredMembers class.